CMakeLists.txt

```diff
@@ -22,7 +23,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
 string(REPLACE "-DNDEBUG" "" CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO}")
 string(APPEND CMAKE_CXX_FLAGS_RELWITHDEBINFO " -g -DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG ")
 string(APPEND CMAKE_CXX_FLAGS_DEBUG " -DSPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_DEBUG ")
-string(APPEND CMAKE_CXX_FLAGS " -Wall -Wextra ")
+string(APPEND CMAKE_CXX_FLAGS " -Wall -Wextra")
+#string(APPEND CMAKE_CXX_FLAGS " -v -lstdc++")
```

Comment:
Looks like a leftover :-) Did you have trouble with compiling rFaaS on machines with libc++?

Reply:
Perhaps my system was weirdly configured, but compilation with clang only worked with explicit linking of stdc++. However, compilation with gcc worked just fine, so I went with that.